internal/ihop/definition.go

```diff
@@ -151,6 +162,14 @@ func NewDefinitionFromFile(path string, unbuffered bool, secrets ...string) (Def
 		definition.Run.Shell = "/sbin/nologin"
 	}
 
+	if definition.SupportURL == "" && strings.Contains(definition.Homepage, "github.com") {
+		definition.SupportURL = fmt.Sprintf("%s/blob/main/README.md", strings.TrimSuffix(definition.Homepage, "/"))
```

Comment:
Did we want to fail if the `SupportURL`/`BugReportURL` are empty and the homepage doesn't contain `github.com`?